pandas/tests/resample/test_base.py

```diff
@@ -0,0 +1,258 @@
+# pylint: disable=E1101
+
+from datetime import datetime, timedelta
+
+import numpy as np
+import pytest
+
+from pandas.compat import range, zip
+from pandas.errors import AbstractMethodError
+
+import pandas as pd
+from pandas import DataFrame, Series
```

Comment:
btw make sure you isort these new files and you may need to remove test_resample from setup.cfg

Reply:
done